### DISCUSSION TITLE
ERROR: Error parsing url: undefined when creating database [docker-compose]

### DISCUSSION THREAD

**Discussion by ItsNiceCraft** [outline/outline#5154] [Self Hosting] [answered]
Hi!
I am currently trying to install outline to a Proxmox LXC Container (Ubuntu 22.04) using docker-compose, however when creating the database using [the command from the docs](https://docs.getoutline.com/s/hosting/doc/docker-7pfeLP5a8t#h-database) the process fails with the following error log:

```
root@notes:/home/outlineinstall# docker-compose run --rm outline yarn db:create --env=production-ssl-disabled
Starting outlineinstall_redis_1    ... done
Starting outlineinstall_postgres_1 ... done
Creating outlineinstall_outline_run ... done
yarn run v1.22.18
$ sequelize db:create --env=production-ssl-disabled

Sequelize CLI [Node: 16.14.2, CLI: 6.4.1, ORM: 6.20.1]

Loaded configuration file "server/config/database.json".
Using environment "production-ssl-disabled".

ERROR: Error parsing url: undefined

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
ERROR: 1
```

I am using [the docker-compose file provided by the docs](https://docs.getoutline.com/s/hosting/doc/docker-7pfeLP5a8t#h-docker-compose) and my docker.env file is empty.
Could anyone help out?

**Comment by astrodad** [comment]
Same issue! How did you resolve this?

**Reply by ItsNiceCraft** [reply]
Not at all unfortunately. Still waiting for an answer.

**Reply by ItsNiceCraft** [reply]
Are you using Proxmox as well?

**Reply by astrodad** [reply]
No, just plain ol docker.

**Reply by fiadigifrank** [reply]
Same issue +1

**Comment by ItsNiceCraft** [comment]
Created issue #5959 because multiple people seem to have this issue.

**Comment by ItsNiceCraft** [comment, marked as answer]
@astrodad @fiadigifrank https://github.com/outline/outline/issues/5959#issuecomment-1751711810
